health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/Inbox.js

```
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { InboxSearchComposer, Loader } from "@egovernments/digit-ui-components";
+import { InboxConfig } from "../../configs/InboxConfig";
+
+const Inbox = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)